src/h3lib/lib/coordijk.c

```diff
@@ -402,6 +403,54 @@ void _ijkRotate60cw(CoordIJK* ijk) {
     _ijkNormalize(ijk);
 }
 
+/**
+ * Rotates indexing digit 60 degrees counter-clockwise. Returns result.
+ *
+ * @param digit Indexing digit (between 1 and 6 inclusive)
+ */
+int _rotate60ccw(int digit) {
+    switch (digit) {
```

Comment:
This is mostly for my info: I'm a little confused here. Given the ASCII diagram on L45, I was expecting this to look like `(digit + 1) % 6`, and was going to leave a note that this would be clearer, but looking through the cases here that's not the case. What exactly is being rotated here, and how does it relate to the diagram on L45?

Reply:
I assume you mean algos.c line 45? That shows the order, but not the value. The value being rotated here is an [indexing digit](https://uber.github.io/h3/#/documentation/core-library/h3-index-representations), same as used in H3Index. Low bit is k, second bit is j, third bit is i.

Reply:
Thanks, makes sense.